examples/hello_world/linux/my_application.cc

```diff
@@ -9,15 +13,15 @@
 
 struct _MyApplication {
   GtkApplication parent_instance;
-  char** dart_entrypoint_arguments;
+  char **dart_entrypoint_arguments;
```

Comment:
😐 

Let's move `examples/platform_channel/windows/.clang-format` to `examples/.clang-format` so that we get standard formatting across all the examples.